experimental/nodejs-kafka/templates/default/app.js

```diff
@@ -0,0 +1,143 @@
+const express = require('express');
+
+module.exports = (options) => {
+  // options.kafka is a required node-rdkafka:
+  // - api docs: https://blizzard.github.io/node-rdkafka/current/
+  // - quick start: https://github.com/Blizzard/node-rdkafka#usage
+  const Kafka = options.Kafka;
+
+  // options.log is a pino logger:
+  // - https://github.com/pinojs/pino/blob/master/docs/api.md#logger
+  const log = options.log;
+
+  // options.app is an express instance, routes can be configured.
+  const app = options.app;
+
+  // KAFKA_BOOTSTRAP_SERVERS can be injected by the Service Binding Operator
+  // - https://github.com/redhat-developer/service-binding-operator
+  const brokers = process.env.KAFKA_BOOTSTRAP_SERVERS || 'localhost:9092'
+
+  log.info('Kafka features: %s', Kafka.features);
+  log.info('Kafka brokers: %s', brokers);
+
+  // Based on:
+  // - https://github.com/ibm-messaging/event-streams-samples/blob/master/kafka-nodejs-console-sample/app.js#L145
+  const config = {
+    // debug: 'all',  // Setting will cause 'event.log' events.
+    'metadata.broker.list': brokers,
+    // 'log.connection.close' : false, // Why?
```

Comment:
@dalelane the event-streams-sample does this, but its not clear why, even after reading the docs at https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md